.travis/.travis_quickstart.sh

```diff
@@ -30,60 +30,75 @@ java -version
 
 # Quickstart
 DIST_BIN_DIR=`ls -d pinot-distribution/target/apache-pinot-*/apache-pinot-*`
-cd ${DIST_BIN_DIR}
+cd "${DIST_BIN_DIR}"
 
 # Test quick-start-batch
 bin/quick-start-batch.sh &
 PID=$!
 
 PASS=0
+
+# Wait for 30 seconds for table to be set up, then at most 5 minutes to reach the desired state
 sleep 30
-for i in $(seq 1 200)
+for i in $(seq 1 300)
 do
-  COUNT_STAR_RES=`curl -X POST --header 'Accept: application/json'  -d '{"sql":"select count(*) from baseballStats limit 1","trace":false}' http://localhost:8000/query/sql | jq '.resultTable.rows[0][0]'`
-  if [[ "${COUNT_STAR_RES}" =~ ^[0-9]+$ ]]; then
-    if [ "${COUNT_STAR_RES}" -eq 97889 ]; then
+  QUERY_RES=`curl -X POST --header 'Accept: application/json'  -d '{"sql":"select count(*) from baseballStats limit 1","trace":false}' http://localhost:8000/query/sql`
+  if [ $? -eq 0 ]; then
+    COUNT_STAR_RES=`echo "${QUERY_RES}" | jq '.resultTable.rows[0][0]'`
+    if [[ "${COUNT_STAR_RES}" =~ ^[0-9]+$ ]] && [ "${COUNT_STAR_RES}" -eq 97889 ]; then
       PASS=1
       break
     fi
   fi
   sleep 1
```

Comment:
how about sleep 2 or 3 seconds instead of just 1 ?

Reply:
Make it 150 iterations and sleep 2. Also updated the script for Github Actions